GSA/digitalgov.gov
# Cloud and infrastructure topic updates

## Summary

Updating cloud and infrastructure topic page with new "topic collection" page format.

https://federalist-466b7d92-5da1-4208-974f-d61fd4348571.sites.pages.cloud.gov/preview/gsa/digitalgov.gov/ss-cloud-and-infrastructure-topic-updates/topics/cloud-and-infrastructure/

### Preview

[Link to Preview]()



### Solution
Added new content and updated format of page.

### How To Test

1. Navigate to "Cloud and infrastructure" topic page
2. Ensure that content on page matches what was approved by Ammie and Toni
3. Ensure that format of page matches new design and that all resources and communities listed have the correct kicker text

---

### Dev Checklist
